src/test/cli/ceph-authtool/cap-invalid.t

```diff
@@ -3,10 +3,10 @@
 
 # TODO is this nice?
   $ ceph-authtool --cap osd 'broken' kring
-  $ ceph-authtool kring --list|grep -P '^\tcaps '
+  $ ceph-authtool kring --list|grep -E '^[^I]caps '
```

Comment:
i never knew that ERE supported the syntax like this! thought it's a character class excluding capital "I"! and seems the spec does not mention this usage, see http://pubs.opengroup.org/onlinepubs/000095399/basedefs/xbd_chap09.html#tag_09_03_05 .